src/schema/v2/artist/__tests__/index.test.js

```diff
@@ -108,25 +120,45 @@ describe("Artist type", () => {
     })
   })
 
-  it("returns the number of auction results for an artist", () => {
+  it("returns the auction results and number of auction results for an artist", () => {
```

Comment:
Good point! I've not added a test for this case because we will hopefully remove this hack very soon. But you're right and I've added another lot to the test to check if it gets filtered out 👍